lib/hmr.js

```diff
@@ -317,12 +322,14 @@ module.exports = (options = {}) => {
     // 2. delete foo.js
     // 3. change: foo.js <- (!) will think foo is not accepted
     //
-    const forget = Object.keys(changed).map(file => {
-      const url = fileUrls[file]
-      delete fileUrls[file]
-      delete changed[file]
-      return url
-    })
+    const forget = Object.keys(changed)
+      .map(file => {
+        const url = fileUrls[file]
+        delete fileUrls[file]
+        delete changed[file]
+        return url
+      })
+      .filter(Boolean)
```

Comment:
Why? Is this fixing an existing bug? Something new?

Reply:
See #14